docs/user/technical-reference/07-20-function-processing-stages.md

```diff
@@ -1,50 +0,0 @@
-# Function Processing Stages
```

Comment:
I dont see why this document was deleted.
Stages Configured and Running are still valid
Please adjust